Src/Workers/pullRequests.php

```
@@ -11,4 +11,4 @@
 $handler = new PullRequestsHandler();
 $healthCheck = new HealthChecks($healthChecksIoPullRequests, GUIDv4::random());
 $processor = new ProcessingManager("pull_requests", $healthCheck, $logger, $logStream);
-$processor->run([$handler, "handleItem"], 60);
+$processor->run([$handler, "handleItem"], const PROCESSING_INTERVAL = 60;);
```

PHPStan (2.2.5)

[error] Syntax error, unexpected T_STRING, expecting ':' on line 14


(phpstan.parse)